Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env groovy
 
 def installBuildRequirements(){
-	def nodeHome = tool 'nodejs-lts-20'
+	def nodeHome = tool 'nodejs-lts'
```

Comment:
better name will be available when https://gitlab.cee.redhat.com/fuse-tooling-qe/cci-jenkins-config/-/merge_requests/97 merged and propagated to the CI